site/cds_rdm/migration/resolver.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
```

Comment:
last comment about the folder naming. I would maybe propose "legacy" or "redirect" instead of "migration", I would like to avoid confusion with the actual migration code (even if not in this repo)